### PR TITLE
Require Maven 3.2.5

### PR DESCRIPTION
**Pull request by slachiewicz**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -268,7 +267,7 @@
       <dependency>
         <groupId>org.apache.maven</groupId>
         <artifactId>maven-plugin-api</artifactId>
-        <version>3.8.4</version>
+        <version>3.2.5</version><!-- Minimum required Maven Version -->
```
Comment (bmarwell) [inline pom.xml:270]: Why 3.2.5 and not 3.1.1?

Comment (slachiewicz) [reply pom.xml:270]: Everywhere in Maven we bump it to 3.2.5

Comment (bmarwell) [reply pom.xml:270]: ok then never mind!

**Review by bmarwell — APPROVED** [review]

**Comment by bmarwell** [comment]
Please go ahead, @slachiewicz! :) Kindly review my PR as well: 
https://github.com/mojohaus/mojo-parent/pull/226

I was thinking about adding more hashes, but SHA-512 is probably good enough.

**Pull request merged**